pkgs/development/tools/haskell/tinc/default.nix

```diff
@@ -7,12 +7,12 @@
 }:
 mkDerivation {
   pname = "tinc";
-  version = "20160511";
+  version = "20160924";
   src = fetchFromGitHub {
-    owner = "sol";
+    owner = "robbinch";
```

Comment:
Having said this, I can create a branch just for this on @sol's tinc repository just for this, as long as he doesn't mind.